setup.py

```diff
@@ -15,5 +15,16 @@
         "nose2"
     ],
     test_suite="nose2.collector.collector",
-    packages=find_packages(exclude=["bin", "*.tests"])
+    packages=find_packages(exclude=["*.tests"]),
+    entry_points={
+        "console_scripts": [
+            "bin.ark_to_records=opennmt.bin.ark_to_records:main",
```

Comment:
Let's improve all script names:

* Remove `bin.`
* Prefix with `onmt-`
* Replace `_` by `-`

e.g. `onmt-ark-to-records`.